### PR TITLE
Added landing and components page with Buttons

### PR DESCRIPTION
**Pull request by haarsh24**

### CODE REVIEW

**Comment by pnchinmay** [comment]
**Preview link isn't working**
Check netlify's error logs shown below👇

image

Try changing those links to `https` and see if it works
Also, avoid leaving unused variables in js files when deploying on netlify. That too, is treated as error

**Comment by haarsh24** [comment]
> **Preview link isn't working** Check netlify's error logs shown below👇
> 
> image
> 
> Try changing those links to `https` and see if it works Also, avoid leaving unused variables in js files when deploying on netlify. That too, is treated as error

Sure will try this

File: Front-page/style.css [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+:root {
+  --primary: blueviolet;
+  --secondary: #0b1120;
+  --font-color: #637084;
+  --white:#fff;
```
Comment (Ashishgupta08) [inline Front-page/style.css:15]: put color keyword for colors
use only one convention for colors - either HEX codes or color names
Suggestion - use HEX codes

File: Front-page/style.css [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+  display: flex;
+  flex-direction: row;
+  padding: 2rem;
+  align-items: center;
+  background-color: var(--secondary);
+  color: var(--primary);
+  /* position: sticky; */
+  top: 0;
+  width: 100%;
+  font-size: larger;
```
Comment (Ashishgupta08) [inline Front-page/style.css:33]: give specific font-sizes

File: Front-page/style.css [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+}
+.usage {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+.installation {
+  text-align: center;
+}
```
Comment (Ashishgupta08) [inline Front-page/style.css:117]: use better class names that show its usage by name

File: Front-page/style.css [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+  padding: 1rem;
+  border-radius: 1rem;
+  font-size: larger;
+  cursor: pointer;
+}
+.btn.primary:hover{
+  background-color: var(--secondary);
+  color: var(--font-color);
+  border: solid 0.5rem var(--primary);
+}
```
Comment (Ashishgupta08) [inline Front-page/style.css:147]: create two separate classes for `btn `and `btn-primary` and add specific styles to each
means add default style of a button to `btn` class and specific style of primary button to `btn-primary` class

File: components-page/components-style.css [hunk 5]
```diff
@@ -0,0 +1,189 @@
...
+  text-decoration: none;
+}
+.nav {
+  color: var(--primary);
+  margin-left: auto;
+  font-size: larger;
+}
+
+.nav:hover {
+  color: red;
```
Comment (Ashishgupta08) [inline components-page/components-style.css:53]: use Variables

File: components-page/components.html [hunk 6]
```diff
@@ -0,0 +1,168 @@
...
+                    transform: scale(1);
+                    overflow: hidden;
+                  "
+                  sandbox="allow-scripts allow-same-origin"
+                >
+                </iframe>
+              </div>
+            </div>
+            <div class="heading btn-heading">Icons Button</div>
+            <div class="btns">
```
Comment (Ashishgupta08) [inline components-page/components.html:111]: this `div` is a **container** for buttons so you can use class name like `btn-container` or `btn-wrapper`

**Review by Ashishgupta08 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**